.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
Can we s/master/main/ while we're cleaning things up for migration?

Reply:
I no longer think this is a good idea. My sense is that using main as default branch has several disadvantages and no tangible advantages.

Reply:
I think the advantage is using more inclusive language, which makes our projects friendlier places. But `main` is also now the default branch name for new github repos, and the one used by Google projects, whose code style we follow in the browser project. And it's shorter to type.

What disadvantages are bothering you? Github Actions not providing a variable for the default branch so one has to hard-code something the rest of the interface treats as dynamic is certainly one of them.

Reply:
I understand that the master/slave nomenclature from databases is problematic, and I'm in favor of changing that, but git's use of master is unrelated as far as I know, so this seems like an overcorrection to me. But perhaps there are specific issues that I'm not aware of?

Years in free software taught me that being inclusive has little to do with making people use the "right" words. I've seen plenty of bad behavior that checked all the "use inclusive language" boxes. I find that being inclusive simply comes down to behaving well, and that goes much deeper than vocabulary, which is a mere symptom of bad behavior.

Besides, I don't want to change the words I use unless there is a good reason to. I understand that this may sound silly and pedantic but I really do believe that it matters. I'm not fundamentally opposed to this; I just haven't heard a good argument yet that would convince me. If I do, I'll change my mind.